field/ShipFieldCreator.cxx

```diff
@@ -50,7 +50,7 @@ FairField* ShipFieldCreator::createFairField()
 	// Initialise field
 	if ( fMagneticField ) {
 		fMagneticField->Init();
-		fMagneticField->Print();
+		fMagneticField->Print("");
```

Comment:
Upstream FairRoot no longer allows calling the Print-function with an empty bracket.